### ISSUE TITLE
Can you provide the data after pretreatment?

### ISSUE THREAD

**Issue by 18010927657** [tebesu/NeuralCitationNetwork#1]
I am a student from anhui university of China. Can you provide the experimental data after extraction?

**Comment by tebesu** [comment]
Here is the preprocessed data I will also update the readme.

https://drive.google.com/open?id=10tdKRxsq7v9-HdjUaFeqmrojKz6gpvcw

Here is some example usage:
```python
dataset = CitationContextDataset('data/seq_data.h5py')
encoder_vocab = dataset.getEncoderVocab()
decoder_vocab = dataset.getDecoderVocab()
papers = dataset.getPaperClusters()

def getEncoderSequence(sequence):
     return [encoder_vocab.reverse(token) for token in sequence if token != 0]
    
def getDecoderSequence(sequence):
     return [decoder_vocab.reverse(token) for token in sequence if token != 0]

print "Papers", len(papers)
test_index = dataset.getTestIndex()
context = dataset.getContext(test_index[0])

print ' '.join(getDecoderSequence(context['title']))
print ' '.join(getDecoderSequence(context['context']))
print papers[context['citing_cluster']]
```

**Comment by 18010927657** [comment]
Thank you for your reply. I have applied on the link below. Can you give me a permission?

https://drive.google.com/open?id=10tdKRxsq7v9-HdjUaFeqmrojKz6gpvcw

**Comment by 18010927657** [comment]
ok！thanks for your help

**Comment by 18010927657** [comment]
Can you give me your pretreated code and several sets of code for comparison experiment?

**Comment by tebesu** [comment]
I do not understand what you are asking for. Could you please elaborate?

**Comment by 18010927657** [comment]
You say, "A fter preprocessing invalid entries, we obtain 4,549,267 context pairs with 855,735.
Papers in a citations."    I wish to knew
1.  This 4549267 context pairs is how to select.The complete dataset of RefSeer is much larger than this one.
2. Can you provide the code of your baselines? 
Thank you!

**Comment by tebesu** [comment]
I loaded the data into MongoDB. Below are my notes while I performed the preprocessing. It should be able to replicate the preprocessing from the RefSeer to the raw data.
[NCNNotes.pdf](https://github.com/tebesu/NeuralCitationNetwork/files/1933297/NCNNotes.pdf)

I used StanfordNLP to tokenize and lemmatize the text. 

For the Citation Translation Model I used https://github.com/moses-smt/giza-pp
For BM-25 I used Solr
